src/octopeer-github/test/resources/scroll.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en" class=" is-copy-enabled is-u2f-enabled">
```

Comment:
What if I told you this is GitHub fault? To stay consistent with their sources, I'd say we leave it like that